lib/widgets/common.dart

```diff
@@ -102,14 +102,14 @@ mixin CommonWidgetMixin {
       ),
       child: Row(
         mainAxisSize: MainAxisSize.min,
-        crossAxisAlignment: CrossAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.center,
         children: [
           Text(
             tag,
             style: TextStyle(
               fontWeight: FontWeight.w400,
               fontSize: 14.0,
-              height: 1.57,
+              // height: 1.57,
```

Comment:
If this is not required, why don't you remove this line?

Reply:
Thanks, I removed the line.